lib/cli/src/generators/configure.ts

```diff
@@ -20,7 +18,7 @@ function configureMain(addons: string[], custom?: any) {
 function configurePreview(framework: SupportedFrameworks) {
   const parameters = `
 export const parameters = {
-  actions: { argTypesRegex: "^on.*" },
+  actions: { argTypesRegex: "^on[A-Z].*" },
```

Comment:
do you think it would be nice to add a small comment telling people that they can change this to other stuff like `"^(on|handle)[A-Z].*"` in case they use a different pattern, or even already just add that regex because it might useful anyway?